### ISSUE TITLE
unwanted information in api docs

### ISSUE THREAD

**Issue by jabr9983** [apidoc/apidoc#281]
So I have my docs created but I have information that is being displayed that I really don't need or want, the image I attached shows what I am talking about. I only need the Tasks part. How do I remove this  unwanted information though?

![screen shot 2015-06-11 at 10 05 27 am](https://cloud.githubusercontent.com/assets/10716758/8111866/6eef7dde-1021-11e5-83a7-b4be3933c892.png)

**Comment by liko28s** [comment]
Hi, you can set Groups in your [apidoc.json](http://apidocjs.com/#configuration), an then add [@apiGroup](http://apidocjs.com/#param-api-group) in the definition of your TASK api

**Comment by jabr9983** [comment]
that will get rid of those files because they shouldn't be apart of the documentation they are merely for running the api.

**Comment by liko28s** [comment]
i.e apidoc.json file:
```
{
"name": "YOUR API NAME",
"version": "0.1.0",
"description": "YOUR API DESCRIPTION",
"url": "https://your/api/url/",
"groups" : ["TASK", "ANOTHERGROUP"],
"template": {"withCompare" : true,"withGenerator": true}
}
```
in your api definition write
```
/**
* @api {get} /api/url  requestTaskInformationById 
* @apiName requestTaskInformationById 
* @apiDescription SOME DESCRIPTION
* @apiGroup TASK
* @apiParam {String} ........
* ...
*/
```

**Comment by jabr9983** [comment]
Okay thank you

**Comment by jabr9983** [comment]
So that organizes it, but those files are still there even though I am excluding them when I run apidoc, is there a reason why this is happening?

**Comment by KrisSiegel** [comment]
I'm having this exact same issue. Unfortunately this issue was closed without a resolution to the problem so I am opening a new issue and will reference this ticket.

**Comment by KrisSiegel** [comment]
Just for reference if someone else lands here: the fix is documented under #109. Basically ```-e``` is expecting a regex, not a path, and is currently not documented.
